### ISSUE TITLE
Incomplete arc

### ISSUE THREAD

**Issue by zsoltk** [LarsWerkman/Lobsterpicker#9]
I'm trying to implement my own ColorAdapter, and I found that sometimes the arc is incomplete. I'm guessing this is because in the implementation of `LobsterPicker.getAngle(int position)` `segmentWidth` is calculated using integer division:

``` java
int segmentWidth = 360 / nbOfSegments;
```

So when my adapter has 16 colors, 360 / 16 = 22.5 is treated as 22.

When I reduce the number of colors to 15 (360 / 15 = 24), the arc is complete.

See a screenshot [here](http://i.imgur.com/E0YeDmC.png).

Thanks for the great library!

**Comment by LarsWerkman** [comment]
@zsoltk Yes I think that's the issue, Currently I'm totally out of time to fix this issue. A pull request is much appreciated.
